### ISSUE TITLE
Accessiblity

### ISSUE THREAD

**Issue by mmaszkiewicz** [emotionLoop/visualCaptcha#31]
There are 2 accessibility problems:
- Images that are not accessible to a blind person are being read-out by screen readers as internal links (aria-hidden would fix this).
- Once you click on the "Audio challenge" icon new input is being inserted, but it appears in front of it. When using a screen reader you would have to tab through the whole form again to get to that input. Reversing the order to "Audio challenge", "Reload", "Input for the Audio challenge" would allow a blind person to tab through to the end of the form and submit it. It would make it possible to reload a challenge without having to go back.

**Comment by BrunoBernardino** [comment]
Hi @mmaszkiewicz !

Thank you so much for the suggestions here, they sound great!

Let me know what are you thinking for the second change.

The main repo to PR to would be https://github.com/emotionLoop/visualCaptcha-frontend-core and I'm happy to take care of updating the other ones with the final built files, but if you want to do that, you're also welcome ;)
